src/helpers.rs

```diff
@@ -51,6 +55,12 @@ fn cgroup_v2_is_snap(cgroups: &str) -> bool {
         .any(|x| x.unwrap_or(false))
 }
 
+pub(crate) fn dup_to_owned_fd(fd: &impl AsRawFd) -> Result<OwnedFd, Error> {
```

Comment:
Maybe make this return a zvariant::FD?

Reply:
The point of using OwnedFd is that it will automatically be closed on drop and will correctly do the duplication with `F_DUPFD_CLOEXEC`, FD doesn't and won't do this (their OwnedFd does close on Drop tho)

Reply:
I am just not sure why such helper exists and yet all the times we use it we wrap it in a zvariant type. So I wonder if we could do things slightly differently

Reply:
well, ideally zbus learns how to serialize `OwnedFd` and `BorrowedFd<'_>`, no need to turn them into `zbus::Fd` or `zbus::OwnedFd`